pyext/src/rmsd_calculation.py

```diff
@@ -195,6 +209,20 @@ def get_rmfs_coordinates_one_rmf(path, rmf_A, rmf_B, subunit_name=None,
     # Get selection
     if subunit_name:
         s0 = IMP.atom.Selection(h, resolution=1, molecule=subunit_name)
+    elif selection:
+        s0=None
+        for idx,selected_range in enumerate(selection.values()):
+            if idx==0:
+                s0 = IMP.atom.Selection(h, resolution=1,
+                        molecule=str(selected_range[0][2]), 
+                        residue_indexes=range(selected_range[0][0], 
+                        selected_range[0][1]))
+            else:
+                s1 = IMP.atom.Selection(h, resolution=1,
+                        molecule=str(selected_range[0][2]), 
+                        residue_indexes=range(selected_range[0][0], 
+                        selected_range[0][1]))
+                s0 = s0 | s1
     else:
```

Comment:
This is just cut-and-paste from the other function above. Pull this out into a separate function so as not to duplicate code.

Reply:
Moved to a function